Microsoft.Identity.Web/Client/TokenCacheProviders/Session/MSALPerUserSessionTokenCacheProvider.cs

```diff
@@ -24,27 +24,31 @@
 
 using Microsoft.AspNetCore.Http;
 using Microsoft.Identity.Client;
+using System;
 using System.Diagnostics;
 using System.Security.Claims;
 using System.Threading;
 
 namespace Microsoft.Identity.Web.Client.TokenCacheProviders
 {
     /// <summary>
-    /// This is a MSAL's TokenCache implementation for one user. It uses Http session as a backend store
+    /// This is a MSAL's TokenCache implementation for one user. It uses Http session as a back end store
     /// </summary>
     public class MSALPerUserSessionTokenCacheProvider : IMSALUserTokenCacheProvider
     {
         private static ReaderWriterLockSlim SessionLock = new ReaderWriterLockSlim(LockRecursionPolicy.NoRecursion);
 
         /// <summary>
-        /// Once the user signes in, this will not be null and can be ontained via a call to ClaimsPrincipal.Current
+        /// Once the user signs in, this will not be null and can be obtained via a call to ClaimsPrincipal.Current
         /// </summary>
+        // brentsch - why is this internal?
         internal ClaimsPrincipal SignedInUser;
 
         /// <summary>
         /// The HTTP context being used by this app
+        // brentsch - this is confusing as an HttpContext is per http request.
```

Comment:
But this is what allows me access to the httpContext.Sessions collection, which is assume lasts for the user's session?